src/js/sdk.tsx

```diff
@@ -149,9 +149,7 @@ export function init(passedOptions: ReactNativeOptions): void {
 /**
  * Inits the Sentry React Native SDK with automatic instrumentation and wrapped features.
  */
-// Deprecated in https://github.com/DefinitelyTyped/DefinitelyTyped/commit/f1b25591890978a92c610ce575ea2ba2bbde6a89
-// eslint-disable-next-line deprecation/deprecation
-export function wrap<P extends JSX.IntrinsicAttributes>(
+export function wrap<P extends Record<string, unknown>>(
```

Comment:
Hm, we probably want to infer the return type. Let me get back to this. In an hour or so.

Reply:
I thought about that too, but so far no one complained about that. But the `JSX.IntrinsicAttributes` forced users to specify a `key` attribute in the props.